### ISSUE TITLE
Broken on Doom in daemon mode after #57 

### ISSUE THREAD

**Issue by real-or-random** [LionyxML/auto-dark-emacs#75]
After #57, `enable-theme` is used instead of `load-theme` when the theme has already been loaded. This breaks stuff on Doom because Doom relies on advising `load-theme` to keep track of the currently active theme.

See my analysis here: https://github.com/doomemacs/doomemacs/issues/8119 I suggest keeping the discussion there, to keep it in one place.

cc @sellout

**Comment by LionyxML** [comment]
@real-or-random / @sellout  , thanks for all your effort in this issue.

I think it is safe to mark it as closed, right?

**Comment by sellout** [comment]
It seems that way to me, but I don’t use Doom, so I defer to @real-or-random.

**Comment by real-or-random** [comment]
Yes, I can confirm that the changes to doom emacs have fixed the problem.
